.clang-format

```diff
@@ -3,7 +3,8 @@ Language: Cpp
 BasedOnStyle: LLVM
 AlwaysBreakTemplateDeclarations: Yes
 BreakBeforeBraces: Attach
-ColumnLimit: 160
+ColumnLimit: 120
+=======
```

Comment:
What's the `===` for?

Reply:
Sorry was leftover from github modifying it. Please remove.

Reply:
Done :).